### ISSUE TITLE
[explicit-member-accessibility] Override off not respected for constructor parameter properties

### ISSUE THREAD

**Issue by travigd** [typescript-eslint/typescript-eslint#617]
**Repro**

```js
{
  "rules": {
    "no-parameter-properties.md": "off",
    "@typescript-eslint/explicit-member-accessibility": ["error", {
      accessibility: "no-public",
      overrides: {
        constructors: "off",
      }
    }],
  }
}
```

```TS
class Foo {
  constructor(public bar) { ... }
}
```

**Expected Result**
No error.

**Actual Result**
Error.
```
error    Public accessibility modifier on parameter property foo
@typescript-eslint/explicit-member-accessibility
```

**Additional Info**

**Versions**

```
    "@typescript-eslint/eslint-plugin": "^1.10.2",
    "@typescript-eslint/parser": "^1.10.2",
    "typescript": "^3.5.1",
    "eslint": "^5.16.0",
```

Was able to work around by setting default to off and overriding on for properties and methods.

**Comment by bradzacher** [comment]
Taking the time to review this issue properly.
I don't think this is correct.

The `constructor` option disables checks on the constructor only.
If you also want to turn off checks for parameter properties, you should also use the `parameterProperties` option.
